root/report/VotingReportsIndex.js

```diff
@@ -0,0 +1,363 @@
+           are correct, but do not know where to start, the following reports
+           should be useful. These are pre-defined {edit_search|edit searches};
+           once you're comfortable with the edit search you can refine them
+           further or just create your own personal reports for your own use!`,
+          {edit_search: '/search/edits'},
+        )}
+      </p>
+
+      <p>
+        {exp.l(
+          `While reviewing the work of your fellow editors, always keep the
+           {coc|Code of Conduct} in mind. Almost all editors want to help, so
+           your goal as a voter is to help them help better. This applies even
+           more for beginners: always try to be helpful and patient with them,
+           even if they are making mistakes, so that they'll hopefully grow into
+           better editors! That said, if you find an editor that seems to be
+           vandalizing the data, you can always report them from their profile.`,
+          {coc: '/doc/Code_of_Conduct'},
+        )}
+      </p>
+
+      <h2>{l('Destructive edits')}</h2>
+      <p>
+        {l(
+          `Destructive edits (removals and merges) are often very hard or even
+           impossible to revert. As such, an incorrect destructive edit that
+           applies unnoticed can cause quite a big mess! Most are guaranteed
+           to remain open for at least two full days even if they get three
+           Yes votes, to avoid them closing too quickly, but it's always good
+           to get more eyes on them. Below you can find four different searches:
+           one for all destructive edits (which might be overwhelming sometimes),
+           one for entity merges and removals only (the edits more likely to
+           cause a mess if they incorrectly go through), one for relationship
+           removals only, and one for destructive changes to releases (track,
+           medium and release label removals).`,
+        )}
+      </p>
+      <ul>
+        <VotingReportRow
+          noOwnEditsUrl="/search/edits?auto_edit_filter=&order=asc&negation=0&combinator=and&conditions.0.field=type&conditions.0.operator=%3D&conditions.0.args=9&conditions.0.args=84&conditions.0.args=4&conditions.0.args=153&conditions.0.args=134&conditions.0.args=14&conditions.0.args=64&conditions.0.args=74&conditions.0.args=24&conditions.0.args=225%2C223%2C311&conditions.0.args=143&conditions.0.args=44&conditions.0.args=83&conditions.0.args=3&conditions.0.args=315&conditions.0.args=152&conditions.0.args=133&conditions.0.args=78&conditions.0.args=410&conditions.0.args=13&conditions.0.args=53&conditions.0.args=63&conditions.0.args=73&conditions.0.args=23&conditions.0.args=36&conditions.0.args=224&conditions.0.args=142&conditions.0.args=211&conditions.0.args=43&conditions.0.args=47&conditions.1.field=status&conditions.1.operator=%3D&conditions.1.args=1&conditions.2.field=editor&conditions.2.operator=not_me&conditions.2.name=&conditions.2.args.0="
```

Comment:
I'd mainly just like to see something done about these URLs if possible. I guess splitting them all across multiple lines would also be horrible and make the file 10,000 lines long, but...maybe there's an alternative. For example, all the `noOwnEditsUrl` URLs probably have a common set of conditions and other parameters to filter out own edits. Perhaps you could extract the common parts of these different URLs into constants, and concatenate them where needed? (So any common conditions included in a constant would have to start with `conditions.0` and end at `conditions.n` for some `n`, and whatever strings use the constant would have to have their other conditions all start at `n + 1`.) This also sounds really tedious though and perhaps not a good use of dev time. So if you don't think any of that is worth the effort I guess we can include it as-is.

Reply:
I hate the URLs here as much as you do, but tbh I think this should be fixed by fixing our stupid edit search URLs. If you have any idea how to make them less absurdly long, I'd be interested :p Your suggestion would make a lot of sense to me if it wasn't because of the condition numbering, but once it starts requiring modifying the constant to allow for changing condition numberings, it stops sounding like it saves effort / helps readability...

Reply:
Yeah, perhaps we should add some shortcut parameters for common conditions, like "me_no_vote=1" (expect a better name and example, lol). But something we can improve on later.

Reply:
Maybe it would be more readable to split by conditions at least, for this example:
```suggestion
          noOwnEditsUrl="/search/edits?" +
          "conditions.0.field=type&conditions.0.operator=%3D&conditions.0.args=9&conditions.0.args=84&conditions.0.args=4&conditions.0.args=153&conditions.0.args=134&conditions.0.args=14&conditions.0.args=64&conditions.0.args=74&conditions.0.args=24&conditions.0.args=225%2C223%2C311&conditions.0.args=143&conditions.0.args=44&conditions.0.args=83&conditions.0.args=3&conditions.0.args=315&conditions.0.args=152&conditions.0.args=133&conditions.0.args=78&conditions.0.args=410&conditions.0.args=13&conditions.0.args=53&conditions.0.args=63&conditions.0.args=73&conditions.0.args=23&conditions.0.args=36&conditions.0.args=224&conditions.0.args=142&conditions.0.args=211&conditions.0.args=43&conditions.0.args=47&" +
          "conditions.1.field=status&conditions.1.operator=%3D&conditions.1.args=1&" +
          "conditions.2.field=editor&conditions.2.operator=not_me"
```

I also removed unneeded defaults.